azkaban-common/src/main/java/azkaban/ramppolicy/SimpleAutoRampPolicy.java

```diff
@@ -22,76 +22,51 @@
 
 
 /**
- * Simple Auto Ramp Policy will be divided to 4 stages
+ * Simple Auto Ramp Policy will be divided to 5 stages
  *  stage 1: 5%
  *  stage 2: 20%
  *  stage 3: 50%
- *  stage 4: 100%
+ *  stage 4: 75%
+ *  stage 5: 100%
  */
-public class SimpleAutoRampPolicy extends AbstractRampPolicy {
-  private static int RAMP_STAGE_MAX = 4;
+public class SimpleAutoRampPolicy extends SimpleRampPolicy {
   private static int ONE_DAY = 86400;
   public SimpleAutoRampPolicy(Props sysProps, Props privateProps) {
     super(sysProps, privateProps);
   }
 
-  @Override
-  public boolean check(
-      ExecutableFlow flow,
-      ExecutableRamp executableRamp
-  ) {
-    if (executableRamp.getState().isPaused()) {
-      return false;
-    }
 
-    updateForRampUp(executableRamp, RAMP_STAGE_MAX, ONE_DAY);
+  @Override
+  protected int getMaxRampStage() {
+    return 5;
```

Comment:
Can we use enums or constants instead of hardcoded numbers here?
Also, making RAMP_STAGE_MAX protected should be good enough and avoids having this function.

Reply:
copied.